### ISSUE TITLE
Casting errors in edslib_python_buffer.c when compiling on a 32-bit Raspberry Pi

### ISSUE THREAD

**Issue by dmccomas** [jphickey/cfe-eds-framework#8]
See the following compilation output:

![2022-05-13-053106_1519x929_scrot](https://user-images.githubusercontent.com/1598728/168257005-994543c0-f757-4bf5-8481-4e8cfa01a4f6.png)

**Comment by jphickey** [comment]
Yeah, these are bogus warnings, notably some of them are actually in the Python headers, not here (i.e. in the definition of the `PyObject_New` macro).

One workaround might be to use `void` instead of the real/intended pointer type.  C will allow a `void*` to be implicitly cast to any other pointer type and won't issue a warning about it.
